membership-attribute-service/app/actions/AuthAndBackendViaAuthLibAction.scala

```diff
@@ -22,7 +22,7 @@ class AuthAndBackendViaAuthLibAction(
       case Left(AuthenticationFailure.Unauthorised) => Left(Results.Unauthorized)
       case Left(AuthenticationFailure.Forbidden) => Left(Results.Forbidden)
       case Right(authenticatedUser) =>
-        val backendConf = if (isTestUser(authenticatedUser.username)) {
+        val backendConf = if (testUserChecker.isTestUser(authenticatedUser.primaryEmailAddress)(authenticatedUser.logPrefix)) {
```

Comment:
now when a test user is detected, it is written to the logs together with its identity id